packages/models-library/src/models_library/basic_regex.py

```diff
@@ -75,3 +75,5 @@
     r"(?::(?P<docker_tag>[\w][\w.-]{0,127}))?"
     r"(?P<docker_digest>\@sha256:[a-fA-F0-9]{32,64})?$"
 )
+
+PROPERTY_KEY_RE = r"^[-_a-zA-Z0-9]+$"  # TODO: PC->* it would be advisable to have this "variable friendly" (see VARIABLE_NAME_RE)
```

Comment:
forgotten todo, and it is actually json key friendly (at least preventing invalid ones, I guess there are plenty of RE available for these)